guide/payments/receive.md

```diff
@@ -0,0 +1,219 @@
+The simplest way to receive bitcoin is by generating and sharing just a Bitcoin address with the sender. Receivers can share the address by either showing it as a scannable QR Code or as plain text that can be shared via any regular communication tool like email, instant messaging, SMS, etc.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/simple-share.png"
+   retina = "/assets/images/guide/payments/receive/simple-share@2x.png"
+   alt-text = "Screen showing how receiver is able to share a bitcoin address as soon as they enter the request flow"
+   width = 250
+   height = 541
+   layout = "float-right-desktop"
+%}
+
+</div>
+
+There are multiple types of addresses. Sharing one that is incompatible with the sender's wallet application can prevent a successful payment. Read more about addresses such [address compatibility]({{ '/guide/glossary/#address-compatibility' | relative_url }}) issues in the glossary.
+
+---
+
+Many Bitcoin wallet applications focus on providing minimalized interfaces for requesting payments. This is convenient in the short-term as it can facilitate fast and easy payments but may not be in the user’s best interest longer-term. Lists of transactions without information about who they are from and their purpose make it hard to manage personal finances and future transaction privacy.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/transactions-no-label-context.png"
+   retina = "/assets/images/guide/payments/receive/transactions-no-label-context@2x.png"
+   alt-text = "Screen showing a list of transactions with no context just amounts and transaction IDs"
+   caption = "When no additional information is gathered from the receiver, only what is stored on the blockchain can be shown."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/transactions-with-label-context.png"
+   retina = "/assets/images/guide/payments/receive/transactions-with-label-context@2x.png"
+   alt-text = "Screen showing a list of transactions with details about the payment"
+   caption = "While more information can be helpful it requires manual input. Ask which are priorities, and how can you make their entry easier?"
+   width = 250
+   height = 541
+%}
+
+</div>
+
+Since the Bitcoin blockchain only stores a limited amount of payment information, such as the amount and addresses involved in a transaction, any additional information must be stored in your application. This additional information can be vital in providing context to wallet owners on their spending habits and payment histories. It is possible for receivers to share some of these extra details with senders by encoding them in [payment links]({{ '/guide/foundations/wallet-interoperability/' | relative_url }}). Either way, these details have to be manually inputted when generating a payment request.
```

Comment:
```suggestion
Since the Bitcoin blockchain only stores a limited amount of information, such as the amount and addresses involved, any additional information must be stored in your application. This additional information can be vital in providing context to wallet owners on their spending habits and payment histories. These details have to be manually added. It is possible for receivers to share some of these extra details with senders by encoding them in [payment links]({{ '/guide/foundations/wallet-interoperability/' | relative_url }}).
```